### ISSUE TITLE
Korean translation for error TS6133

### ISSUE THREAD

**Issue by sunkibaek** [Microsoft/TypeScript#19947]
**Code**

```ts
class Hello {
  private someProperty: string;

  constructor() {
    this.someProperty = 'hello';
  }
}
```

and then running this in Korean locale with `noUnused*` options on:

```
$ tsc  --locale ko-KR
```

**Expected behavior:**
Proper error messages in Korean.

> translate.ts(2,11): error TS6133: 'someProperty'이(가) 선언되었지만 해당 값이 읽혀진 적이 없습니다.

**Actual behavior:**
Somewhat misleading error messages in Korean.

> translate.ts(2,11): error TS6133: 'someProperty'이(가) 선언되었지만 해당 값을 읽을 수 없습니다.

This reads more like 'the compiler was not able to read the value'.

**Comment by mhegazy** [comment]
//cc @danyeh

**Comment by lukaschal** [comment]
On linguistic review.

**Comment by lukaschal** [comment]
Linguistic answer:

The current target style sounds more natural but for the reason mentioned by the subsidiary of being ambiguous/misleading, the active voice can be changed to the passive.

'~이 읽혀진 적이 없습니다.' part (of the suggestions) however, can be modified to '~은 읽히지 않습니다.'. 

Source: '{0}' is declared but its value is never read.
New TGT: '{0}'이(가) 선언되었지만 해당 값은 읽히지 않습니다.

Source: Property '{0}' is declared but its value is never read.
New TGT: 속성 '{0}'이(가) 선언되었지만 해당 값은 읽히지 않습니다.

Affected file: diagnosticMessages.generated.json

**Comment by sunkibaek** [comment]
@lukaschal 

I can see the good progress but may I suggest another:

> '{0}'이(가) 선언되었지만 해당 값은 읽히지 않습니다

The sentence above simply does not mean what the original message meant to say. It does not have the nuances that it has NEVER been read before. It reads more like it CANNOT be read as of writing. (More like 'I am trying to read this but now I know I cannot read it for some reason') In that sense my new suggestion is

> '{0}'이(가) 선언은 되었지만 해당 값이 읽히지는 않았습니다

Here I have made three changes:

- Using '않았습니다' instead of '않습니다': '않았습니다' gives better historical nuance that it has ever, never been read before while '않습니다' describes current inability to do a certain type of action.
- Adding '은/는' to the end of '선언/읽히지' will give better context of comparison between declaration and reading. Now we are saying more clearly the variable needs to be DECLARED and needs to be READ to get rid of this error.
- Changing to '해당 값이': '은/는' indicates the word is the subject of the sentence but it also carries the nuance of comparison. We are not comparing variable A to its value here. It is the act of declaration and the act of reading that matters.

Thanks!

**Comment by lukaschal** [comment]
On review.

**Comment by lukaschal** [comment]
New translation ( '{0}'이(가) 선언은 되었지만 해당 값이 읽히지는 않았습니다. ) implemented.
